### ISSUE TITLE
[calendar] column all_day in postgresql needs to be boolean

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#305]
```
What steps will reproduce the problem?
1. roundcube 0.7.1 with bundle calendar plugin
2. database postgresql


What is the expected output? What do you see instead?
No synchro happens


What version of the product are you using? On what operating system?
roundcube 0.7.1 with bundle calendar plugin

Please provide any additional information below.
changing all_day from smallint to boolean in "events", "events_cache", 
"events_caldav"
```

Original issue reported on code.google.com by `gianluca...@gmail.com` on 5 Mar 2012 at 3:13

**Comment by GoogleCodeExporter** [comment]
```
I don't use Postgresql.

Is this the correct syntax in the postgresql script?

all_day boolean DEFAULT FALSE,


```

Original comment by `rol...@liebl.ath.cx` on 6 Mar 2012 at 8:51
- Changed state: **Reproducing**

**Comment by GoogleCodeExporter** [comment]
```
Better:
all_day boolean NOT NULL DEFAULT false

great job!
G
```

Original comment by `gianluca...@gmail.com` on 6 Mar 2012 at 5:34

**Comment by GoogleCodeExporter** [comment]
```
P.S.
feel free to use me as beta tester for postrgres, I'll be glad to help you
with this plugin :-)

G



2012/3/6 Gianluca Giacometti <gianlucagiacometti68@gmail.com>
```

Original comment by `gianluca...@gmail.com` on 6 Mar 2012 at 7:05

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 9 Mar 2012 at 7:47
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Issue 306 has been merged into this issue.
```

Original comment by `rol...@liebl.ath.cx` on 18 Mar 2012 at 4:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 18 Mar 2012 at 4:35
- Changed state: **Re-Opened**

**Comment by GoogleCodeExporter** [comment]
Original comment by `rol...@liebl.ath.cx` on 21 Mar 2012 at 2:46
- Changed title: **[calendar] column all_day in postgresql needs to be boolean**

**Comment by GoogleCodeExporter** [comment]
```
I'll remove all_day field from database. It is not really needed. 'All day' is 
already calculated by event duration respectively events without an end time 
are treated as all day.
```

Original comment by `rol...@liebl.ath.cx` on 22 Mar 2012 at 3:34

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 8.0 - 22.03.2012. 'allDay' field is redundant now and can be removed 
from database tables.

NOTE: You need 
[http://code.google.com/p/myroundcube/downloads/detail?name=plugin_manager.zip 
plugin_manager] to fetch updates.
```

Original comment by `rol...@liebl.ath.cx` on 28 Mar 2012 at 5:08
- Changed state: **Fixed**
